docs/design/mono/mobile-runtimeconfig-json.md

```diff
@@ -0,0 +1,143 @@
+# Mobile runtimeconfig.json host configuration
+
+## Motivation
+
+`runtimeconfig.json` is a file used by .NET 5+ applications to pass arbitrary configuration parameters via the application host to the .NET runtime.  On mobile and WebAssembly, we would like to support passing configuration properties from the Mono embedder to the runtime.
```

Comment:
Do we plan to adopt this on wasm? I thought this was intended to be mobile-specific for now? cc: @marek-safar

Reply:
I think when I was drafting this part "mobile and WebAssembly" was just shorthand for "workloads where Mono is the runtime"

Reply:
Quote from Marek "It should be supported by all Mono workloads, in other words passing runtimeconfig.json settings have to always work"